lib/jobba/status.rb

```diff
@@ -219,7 +223,7 @@ def restart!
     def archive_attempt!
```

Comment:
Usually....one adds a bang method when there's a non-bang method available.  

https://github.com/rubocop-hq/ruby-style-guide#safe-because-unsafe

![image](https://user-images.githubusercontent.com/352161/53435987-ec1e1d00-39ae-11e9-97c3-1c82a9540f8d.png)

Reply:
yeah, I agree with what you're saying. ideally we'd have a non bang version and then have the bang version raise an exception or do something "unsafe".  However the `archive_attempt!` doesn't raise, or really do anything dangerous that I can see.

But since this is a library that's in use I don't think we should change the API without a strong reason.  This doesn't quite rise to that level for me.